external/netstandard/netstandard.depproj

```diff
@@ -22,6 +22,10 @@
       <!-- only for netfx verticals will this project provide runtime assets -->
       <RuntimePath>$(RuntimePath)</RuntimePath>
     </BinPlaceConfiguration>
+    <BinPlaceConfiguration Include="$(Configuration)">
```

Comment:
The one in dir.targets as far as I know will only pick up the source generated bits but not the netstandard facades. e.g whithout this bin place System.Runtime.dll would not be in the testhost netfx folder.